### PR TITLE
fix(Checkbox): disable pointer event on svg

### PR DESCRIPTION
**Pull request by liuderchi**
To diable pointer event on svg elements to prevent trigger dialog `onHide`
- [x] InputCheckbox
- [x] Popover
- [x] Tooltip

### CODE REVIEW

File: src/Popover/Popover.js [hunk 1]
```diff
@@ -65,7 +64,7 @@ class Popover extends React.PureComponent<Props, State> {
 
             {/* Note: Arrow */}
             <Overlay show={isOpen} resize target={target} {...position.arrow}>
-              <Arrow key="arrow" />
+              <StyledArrow key="arrow" />
```
Comment (liuderchi) [inline src/Popover/Popover.js]: Looks like setting `pointer-event` on Arrow CANNOT avoid `onHide` getting called

When in `onHide`, `event.target` will be _container div of `StyledArrow`_, 
which is NEVER children of the dialog DOM.

REF: [storybook: Popover > in dialog](https://deploy-preview-125--mcs-ui.netlify.com/?selectedKind=Popover&selectedStory=in%20dialog&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

File: src/Popover/Popover.example.js [hunk 2]
```diff
@@ -121,4 +123,36 @@ import {
...
       },
     },
+  )
+  .add(
+    'in dialog',
+    () => (
+      <ConfirmDialog
+        show
+        onHide={e => {
+          action('onHide');
```
Comment (evenchange4) [inline src/Popover/Popover.example.js]: `action('onHide')();`

File: src/Popover/Popover.js [hunk 3]
```diff
@@ -64,9 +64,14 @@ class Popover extends React.PureComponent<Props, State> {
             </Overlay>
 
             {/* Note: Arrow */}
-            <Overlay show={isOpen} resize target={target} {...position.arrow}>
+            <StyledOverlay
```
Comment (liuderchi) [inline src/Popover/Popover.js]: NOTE: do not wrap `StyledCard` (L61) with `StyledOverlay`;
let user handles `onHide` by theirselves

File: src/Tooltip/styled-components.js [hunk 4]
```diff
@@ -9,4 +10,8 @@ export const StyledCard: ReactComponentFunctional<CardProps> = styled(Card)`
   color: ${({ theme }: ThemeProps) => theme.color.white};
 `;
 
+export const StyledOverlay: React.ComponentType<*> = styled(Overlay)`
```
Comment (liuderchi) [inline src/Tooltip/styled-components.js]: TODO: import Overlay flowtype

File: src/Popover/styled-components.js [hunk 5]
```diff
@@ -11,4 +12,8 @@ export const StyledCard: React.ComponentType<CardProps> = styled(Card)`
   border: 1px solid ${({ theme }: ThemeProps) => theme.color.grayBase};
 `;
 
+export const StyledOverlay: React.ComponentType<*> = styled(Overlay)`
```
Comment (liuderchi) [inline src/Popover/styled-components.js]: TODO: import Overlay flowtype

File: src/Tooltip/styled-components.js (hunk 4 above)
Comment (evenchange4) [inline src/Tooltip/styled-components.js]:
```
React.ComponentType<OverlayProps> = styled(Overlay)`
```

**Pull request merged**